<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sql</class>
 <widget class="QMainWindow" name="Sql">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="handleWidth">
       <number>5</number>
      </property>
      <widget class="TextEditCompleter" name="textQuery">
       <property name="font">
        <font>
         <family>Courier New</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
      <widget class="QTableView" name="tableData">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="alternatingRowColors">
        <bool>false</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SelectionMode::ExtendedSelection</enum>
       </property>
       <property name="gridStyle">
        <enum>Qt::PenStyle::DotLine</enum>
       </property>
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBarQuery">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>24</width>
     <height>24</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonStyle::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRun"/>
   <addaction name="actionFormat"/>
   <addaction name="actionMacros"/>
   <addaction name="actionFavorites"/>
   <addaction name="actionSave"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonStyle::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCommit"/>
   <addaction name="actionAuto_commit"/>
   <addaction name="actionIncrease"/>
   <addaction name="actionReduce"/>
  </widget>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/resources/square.svg</normaloff>:/icons/resources/square.svg</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="toolTip">
    <string>Run (CTRL+R)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionFormat">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/resources/autobrightness.svg</normaloff>:/icons/resources/autobrightness.svg</iconset>
   </property>
   <property name="text">
    <string>Format</string>
   </property>
   <property name="toolTip">
    <string>Format (CTRL+F)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/resources/heart tick.svg</normaloff>:/icons/resources/heart tick.svg</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save default (CTRL+S)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionFavorites">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/resources/lovely.svg</normaloff>:/icons/resources/lovely.svg</iconset>
   </property>
   <property name="text">
    <string>Favorites</string>
   </property>
   <property name="toolTip">
    <string>Add to favorites (CTRL+Y)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionIncrease">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/resources/search zoom-2.svg</normaloff>:/icons/resources/search zoom-2.svg</iconset>
   </property>
   <property name="text">
    <string>Increase</string>
   </property>
   <property name="toolTip">
    <string>Increase font size</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+=</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionReduce">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/resources/search zoom.svg</normaloff>:/icons/resources/search zoom.svg</iconset>
   </property>
   <property name="text">
    <string>Decrease</string>
   </property>
   <property name="toolTip">
    <string>Decrease font size</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionCommit">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/resources/path.svg</normaloff>:/icons/resources/path.svg</iconset>
   </property>
   <property name="text">
    <string>Commit</string>
   </property>
   <property name="toolTip">
    <string>Commit changed data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionMacros">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/resources/shapes.svg</normaloff>:/icons/resources/shapes.svg</iconset>
   </property>
   <property name="text">
    <string>Macros</string>
   </property>
   <property name="toolTip">
    <string>Macro builder</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="actionAuto_commit">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/icons/resources/rotate left.svg</normaloff>:/icons/resources/rotate left.svg</iconset>
   </property>
   <property name="text">
    <string>Auto commit</string>
   </property>
   <property name="toolTip">
    <string>Auto commit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TextEditCompleter</class>
   <extends>QTextEdit</extends>
   <header>texteditcompleter.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
